cumulus/parachains/runtimes/collectives/collectives-westend/src/fellowship/mod.rs

```diff
@@ -258,9 +254,9 @@ parameter_types! {
 	// Benchmark bond. Needed to make `propose_spend` work.
 	pub const TenPercent: Permill = Permill::from_percent(10);
 	// Benchmark minimum. Needed to make `propose_spend` work.
-	pub const BenchmarkProposalBondMinimum: Balance = 1 * DOLLARS;
+	pub const BenchmarkProposalBondMinimum: Balance = 1 * testnet_parachains_constants::westend::currency::DOLLARS;
 	// Benchmark maximum. Needed to make `propose_spend` work.
-	pub const BenchmarkProposalBondMaximum: Balance = 10 * DOLLARS;
+	pub const BenchmarkProposalBondMaximum: Balance = 10 * testnet_parachains_constants::westend::currency::DOLLARS;
```

Comment:
cleaner if imported on top with `use testnet_parachains_constants::westend`

Reply:
ok, thank you, fixed here: https://github.com/paritytech/polkadot-sdk/pull/3134/commits/406c5bf199a90838bb90dcd23d7bea89eb434ce9